### PR TITLE
Update pandas-cheatsheet.md

### PR DESCRIPTION
**Pull request by ritwikdalmia**

### CODE REVIEW

**Review by crescentpartha — COMMENTED** [review]
Please, give the reference related to this, I didn't find any documentation related to this.

**Comment by ritwikdalmia** [comment]
https://pandas.pydata.org/docs/user_guide/visualization.html  

this all the details has been taken from here i have precise the data from studying here

File: CheatSheets/pandas-cheatsheet.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ created: 2022-10-24
   - [Filter, Sort, and Groupby](#filter-sort-and-groupby)
   - [Join/Combine](#joincombine)
   - [Statistics](#statistics)
+  - [Data Visualization with dataframe](#Data-Visualization-with-dataframe)
```
Comment (crescentpartha) [inline CheatSheets/pandas-cheatsheet.md:21]: So, you actually defined the `terminology and definitions` in this section.

- Data Visualization with Dataframe
    - Terminology and Definitions

Please, add all of your codes under the `Terminology and Definitions` section in this way.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, follow the requirements and modify the codes.

File: CheatSheets/pandas-cheatsheet.md [hunk 2]
```diff
@@ -175,3 +176,55 @@ This cheat sheet will guide through the basics of the Pandas library from the da
...
+|`table `| boolean, Series or DataFrame, default False. If True, draw a table using the data in the DataFrame and the data will be transposed to meet matplotlib’s default layout. If a Series or DataFrame is passed, use passed data to draw a table.|
+|`yerr `| DataFrame, Series, array-like, dict and str. See Plotting with Error Bars for detail.|
+|`xerr `| same types as yerr.|
+|`stacked `| boolean, default False in line and bar plots, and True in area plot. If True, create stacked plot.|
+|`sort_columns `| boolean, default False. Sort column names to determine plot ordering|
+|`secondary_y `| boolean or sequence, default False. Whether to plot on the secondary y-axis If a list/tuple, which columns to plot on secondary y-axis|
+|`mark_right `| boolean, default True. When using a secondary_y axis, automatically mark the column labels with “(right)” in the legend|
+|`kwds`|  keywords .Options to pass to matplotlib plotting method|
+|`axes`|  matplotlib.axes.Axes or numpy.ndarray of them|
+|**king:str**|
```
[the remark this answers is not in the document]
Comment (ritwikdalmia) [reply CheatSheets/pandas-cheatsheet.md:219]: actually it a type of plot like line plot bar plot histogram and all above parameters are same for each type of plot

**Comment by ritwikdalmia** [comment]
resolved all the conflicts

**Review by crescentpartha — APPROVED** [review]
Approved

**Pull request merged**